src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -46,6 +46,22 @@ public void Dispose()
             var server = new TcpListener(endPoint);
             server.Start();
 
+            _clientCertificateRemovedByFilter = false;
+
+            if (PlatformDetection.IsWindows7 && 
+                !useClientSelectionCallback && 
+                !Capability.IsTrustedRootCertificateInstalled())
+            {
+                // https://technet.microsoft.com/en-us/library/hh831771.aspx#BKMK_Changes2012R2
+                // Starting with Windows 8, the "Management of trusted issuers for client authentication" has changed:
+                // The behavior to send the Trusted Issuer List by default is off.
+                //
+                // In Windows 7 the DN list is sent within the Server Hello. If the client selection callback isn't 
+                // used and the client certificate's issuer is not in the server's Trusted Root Authorities the client,
+                // not send the certificate.
```

Comment:
Nit: I think I understand what this is saying, but I feel like a few words got lost after "Trusted Root Authorities".